css/style.css

```diff
@@ -0,0 +1,58 @@
+.heading{
```

Comment:
css folder structure be there containing reset.css, variable.css, common.css , modules. soon i am going to ask to change themes, it should be matter of changing variable.css

Reply:
follow the comments in put on Abida's assignment